indico/web/breadcrumbs.py

```diff
@@ -31,6 +31,11 @@ def render_breadcrumbs(*titles, category=None, event=None, management=False, cat
     assert titles or event or category
     if not category and not event:
         items = [(_('Home'), url_for_index())]
+    elif event and not category and not event.category:
```

Comment:
is the `not category` check actually needed?

Reply:
I check for `category` because of the function signature:

```python
def render_breadcrumbs(*titles, category=None, event=None, management=False, category_url_factory=None):
```

I thought it might be confusing if somebody passes a category in the future and it still renders the unlisted events breadcrumb. But it might be unlikely anyone passes an Event and also a Category.

Let me know if it's best to remove the check despite that.

Reply:
If you never expect a category, maybe `assert category is None` inside the if body would be better? That way it's clear that it's not part of the condition, but if the condition is met we're never supposed to have a category.